examples/gno.land/r/demo/gnofundme/gnofundme.gno

```diff
@@ -0,0 +1,253 @@
+package gnofundme
+
+import (
+	"bytes"
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+	pkg "gno.land/p/demo/gnofundme"
```

Comment:
I think this name `pkg` could be better called to avoid confusion since it's not the only package you're calling. This would help the readability of your code.

Reply:
thanks for the advise, I'll modify it before the final version.